Project.toml

```diff
@@ -28,6 +29,7 @@ StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 StochasticDiffEq = "789caeaf-c7a9-5a7d-9973-96adeb23e2a0"
 SurfaceFluxes = "49b00bb7-8bd4-4f2b-b78c-51cd0450215f"
+TOML = "fa267f1f-6049-4f14-aa54-33bafae1ed76"
```

Comment:
I don't think we need to add this dep

Reply:
Probably better to add to the other envs:
 - test/
 - perf/
 - examples/
 - docs/
This way ClimaAtmos.jl is not shipped with (nor depends on) it